deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java

```diff
@@ -121,14 +121,50 @@ public void postProcess() {
     }
 
     @Override
-    protected String getSymbolName(String input) {
-        for (Entry<String, String> entry : specialCharReplacements.entrySet()) {
-            if (input.startsWith(entry.getKey())) {
-                return input.length() > entry.getKey().length()
-                        ? entry.getValue() + "_" + input.substring(entry.getKey().length())
-                        : entry.getValue() + "_symbol";
+    public String toEnumVarName(String value, String datatype) {
```

Comment:
Im wondering if we can call super and then add

        for (Map.Entry<String, String> entry : this.specialCharReplacements.entrySet()) {
            if (enumValue.contains(entry.getKey())) {
                enumValue = enumValue.replace(entry.getKey(), "_".concat(entry.getValue()).concat("_"));
            }
        }